pkg/cmd/stack/translate.go

```diff
@@ -420,7 +420,7 @@ func translateService(svcName string, s *model.Stack) *apiv1.Service {
 func getSvcPublicPorts(svcName string, s *model.Stack) []model.Port {
 	result := []model.Port{}
 	for _, p := range s.Services[svcName].Ports {
-		if !model.IsSkippablePort(p.ContainerPort) && (p.HostPort != 0 || s.Services[svcName].Public) {
+		if !model.IsSkippablePort(p.ContainerPort) && p.HostPort != 0 {
```

Comment:
This change was made in the `2.3.2` release and was causing the bug so we are reverting it

Reply:
Are we also reverting a fix? Can you link to what the associated issue was. I want to evaluate if we release without that fix or not.

Reply:
We are reverting https://github.com/okteto/okteto/pull/2735 with this change